README.md

```diff
@@ -1,3 +1,73 @@
+=head1 NAME
```

Comment:
Come avevo già osservato, il formato POD non va bene per il file README.md che deve essere in formato Markdown. Ci sono tool come pod2markdown (su CPAN) per fare questo tipo di conversione automatica.